moped-editor/src/views/projects/projectView/ProjectNameEditable.js

```diff
@@ -34,6 +34,7 @@ const useStyles = makeStyles((theme) => ({
   },
   titleDisplayField: {
     display: "flex",
+    cursor: "pointer",
```

Comment:
this is great. would you mind adding an `autoFocus` prop to this component's `<TextField/>` ? 😇 

btw, i was surprised to find that hitting the `enter` key submitted the project name edit form. our project summary fields don't have this behavior, i suspect because we're not using a proper <Form> component? will make a note of this.